doc/cheatsheet.md

```diff
@@ -305,6 +402,23 @@ I don't find these operators particularly useful in practice (with the exception
   * `X` inverts `Y`; it converts a specifically-formatted 3-column stream into a multiple-column stream.
   * The specification for what the input matrix must look like is described above in the `Y` operator.
   
+  
+
+##Matrix Operations
+
+Operations on huge matrices are not entirely `ni`ic, since they may require space greater than memory, whichwill make them slow. However, operators are provided to improve These operations are suited best to 
```

Comment:
A big difference between ni and nfu is that ni lets you operate on large streams of small/intermediate matrices. You do this by using a colspec prefix on the `N` operator.